.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+## Why
+
+[Monday.com Ticket](https://code4community-team.monday.com/boards/XXXXXXXX/pulses/XXXXXXX)
+
+## This PR
+
+## Screenshots
+
+## Verification Steps
```

Comment:
Verification steps?

Reply:
Can we also add some text as instructions on what exactly should go in each section?

Reply:
Yeah, wasn't sure if it's best to include here or explain elsewhere